src/lib/model/OutboundBulkTransfersModel.js

```diff
@@ -15,7 +15,7 @@ const { uuid } = require('uuidv4');
 const StateMachine = require('javascript-state-machine');
 const { MojaloopRequests } = require('@mojaloop/sdk-standard-components');
 const { BackendError } = require('./common');
-
+const shared = require('./lib/shared');
```

Comment:
We should really start using aliases throughout our code

Example package.json config: https://github.com/mojaloop/central-ledger/blob/master/package.json#L29
Example usage: https://github.com/mojaloop/central-ledger/blob/336a0a27e908eedeb0dcf8b171ad8c0edfb4c3d8/test/unit/handlers/bulk/shared/validator.test.js#L7

Reply:
Sure, there are lot of places. May be we need a new story or we can do that as part of the new bulk implementation.